src/assets/js/main.js

```diff
@@ -729,3 +729,18 @@ for (var i = 0; i < pathEls.length; i++) {
     autoplay: true
   });
 }
+
+
+var consolestyle = [
+	'background: linear-gradient(to right, #2970ff, #2ED6A1);',
+	'color: #091440',
```

Comment:
Even though the Javascript console in most browsers is rendered using a monospace font, I would still include monospace in the style here just to be sure the ASCII renders correctly in as many places as possible.